module/TaskManagement/Module.php

```diff
@@ -1,162 +1,163 @@
-<?php
-
-namespace TaskManagement;
-
-use AcMailer\Service\MailService;
-use AcMailer\View\DefaultLayout;
-use TaskManagement\Service\CloseTaskListener;
-use Zend\ModuleManager\Feature\AutoloaderProviderInterface;
-use Zend\ModuleManager\Feature\ConfigProviderInterface;
-use TaskManagement\Controller\MembersController;
-use TaskManagement\Controller\TasksController;
-use TaskManagement\Controller\TransitionsController;
-use TaskManagement\Controller\EstimationsController;
-use TaskManagement\Controller\SharesController;
-use TaskManagement\Controller\StreamsController;
-use TaskManagement\Service\NotifyMailListener;
-use TaskManagement\Service\TransferTaskSharesCreditsListener;
-use TaskManagement\Service\StreamCommandsListener;
-use TaskManagement\Service\TaskCommandsListener;
-use TaskManagement\Service\EventSourcingStreamService;
-use TaskManagement\Service\EventSourcingTaskService;
-use TaskManagement\Controller\RemindersController;
-
-class Module implements AutoloaderProviderInterface, ConfigProviderInterface
+<?php
```

Comment:
Questo file risulta completamente modificato (come puoi verificare in "Files changed" ogni riga è stata sostituita), ma non riesco a capire perché. Quali cambiamenti sono stati apportati?